mender-auth/client/client.cpp

```diff
@@ -27,6 +27,11 @@
 #include <common/expected.hpp>
 #include <common/identity_parser.hpp>
 
+namespace mender {
+namespace auth {
+namespace http_client {
```

Comment:
Maybe `http_client` is a bit much? We might use other transport protocols later. See also [this comment](https://github.com/mendersoftware/mender/pull/1274#discussion_r1234914575).

Reply:
Yeh, you're right, I'll do as we spoke about there, and subspace http, and use it in `auth`.